ckan/tests/legacy/logic/test_action.py

```diff
@@ -130,7 +130,7 @@ def test_02_package_autocomplete_match_name(self):
         assert_equal(res_obj['result'][0]['match_displayed'], 'warandpeace')
 
     def test_02_package_autocomplete_match_title(self):
-        postparams = '%s=1' % json.dumps({'q':'a%20w', 'limit': 5})
```

Comment:
what's the significance of this change?

Reply:
This is how an implementation of NGramTokenized differs in SOLR 3 and 4. newer SOLR was able to found record 'A Wonderful Story' with such query, but v3 can find it only if we are providing at least few symbols from a single word. So I've updated this test and now it's searching by part of the word, rather than by part of the whole phrase